bridge/device/cloud/signIn.go

```diff
@@ -83,7 +82,7 @@ func (c *Manager) signIn(ctx context.Context) error {
 		return err
 	}
 	c.updateCredsBySignInResponse(signInResp)
-	log.Printf("signed in\n")
+	c.logger.Infof("signed in")
```

Comment:
add there `deviceID` because with 100 devices didn't know which devices has been signed.

maybe we can extend interface logger about [With(args ...interface{}) Logger](https://github.com/plgd-dev/hub/blob/ab7a1683f2a60f9d365bc810fdfddc245a0f59ab/pkg/log/log.go#L249)

so device alway call `With("deviceId", {id}")` in `device.New` function.

Reply:
so all logs produced by device will contain deviceId

Reply:
The standard `log.Logger` type I'm using has SetPrefix method, which adds given prefix to each log. I've added `[${id}]` prefix to each device log. The logs now look like this:

```
...
2024/02/02 17:09:36 [290d628c-2940-416c-8f51-9355ce568d3b] signed in
2024/02/02 17:09:37 [290d628c-2940-416c-8f51-9355ce568d3b] cannot write response: cannot write request: connection was closed: context canceled
2024/02/02 17:09:37 [290d628c-2940-416c-8f51-9355ce568d3b] signed off
...
```


If that is not sufficient, then I guess we need to migrate to the zap.Logger used in plgd/hub

Reply:
with this version it is fine